cserver.py

```diff
@@ -217,7 +217,7 @@ def query():
     #check if owner has an active sale record or request
     sales = db.session.query(Sale).filter(Sale.owner == owner).count()
     res = []
-    if sales == 0:
+    if sales == 0 and string != 'ratings':
```

Comment:
We should look for another way to provide ratings data publicly if we want to do that without the user having a bucket. I'm partial to keep this behind an account to avoid abuse and to keep incentives right.